operate/webapp/src/main/java/io/camunda/operate/webapp/rest/dto/ListenerDto.java

```diff
@@ -22,6 +23,8 @@ public class ListenerDto {
   private ListenerEventType event;
   private OffsetDateTime time;
 
+  private Object[] sortValues;
```

Comment:
I would change here the data type to private SortValuesWrapper[] sortValues, to make it consistent with other Dto implementations, see for example: https://github.com/camunda/camunda/blob/main/operate/webapp/src/main/java/io/camunda/operate/webapp/rest/dto/activity/FlowNodeInstanceDto.java#L42